.ado/windows-vs-pr.yml

```diff
@@ -441,15 +447,9 @@ jobs:
           collectDumpOn: onAbortOnly
           vsTestVersion: latest
           otherConsoleOptions: '/blame -- RunConfiguration.TestSessionTimeout=300000'
+        # Suspected debug assert in TestRunner hanging tests randomly. Run only on Release for now.
         condition: and(succeeded(), ne(variables['BuildConfiguration'], 'Debug'))
```

Comment:
Does the configuration need to be changed to release?

Reply:
Approving for now, but please follow up here

Reply:
The title is outdated.
Debug can't be re-enabled because of an issue with the TestRunner (unexpected asserts happen on CI runs, which make the whole job hang).